server/dive_tasks/tasks.py

```diff
@@ -611,46 +623,23 @@ def extract_zip(self: Task, folderId: str, itemId: str):
             f"/item/{str(item['_id'])}?folderId={str(created_folder['_id'])}",
         )
 
-        if len(top_level_folders) > 0:
-            for top_folder in top_level_folders.keys():
-                base_folder = ''  # default for no subfolder
-                # single flat/dataset import
-                if len(top_level_folders) != 1:
-                    base_folder = top_folder
-                # multiple flat/dataset import
-                if top_level_folders[top_folder] == 'flat':
-                    utils.upload_zipped_flat_media_files(
-                        gc,
-                        manager,
-                        folderId,
-                        Path(f"{_working_directory}/{top_folder}"),
-                        base_folder,
-                    )
-                # multiple flat file exports combined
-                elif top_level_folders[top_folder] == 'dataset':
-                    utils.upload_exported_zipped_dataset(
-                        gc,
-                        manager,
-                        folderId,
-                        Path(f"{_working_directory}/{top_folder}"),
-                        base_folder,
-                    )
-
-        else:  # flat zip file with media and/or annotation data inside
-            utils.upload_zipped_flat_media_files(gc, manager, folderId, Path(_working_directory))
-
-        # remove auxilary folder and metadata if there are sub folders
-        if len(top_level_folders) > 1:
-            sub_folders = list(gc.listFolder(folderId))
-            aux_folders = list(
-                (x for x in sub_folders if x['name'] == constants.AuxiliaryFolderName)
-            )
-            if len(aux_folders) > 0:
-                gc.delete(f"folder/{aux_folders[0]['_id']}")
-            # remove metadata
-            metdata = [constants.TypeMarker, constants.FPSMarker, constants.DatasetMarker]
-            gc.sendRestRequest(
-                "DELETE",
-                f"folder/{folderId}/metadata",
-                json=metdata,
-            )
```

Comment:
The removal of this means that top level folders have residual metadata that stick around including `fps` and a `type: zip`.  The type may be fine as long as we don't ever use `type` for searching for data.  The `fps` is relative for creating a single dataset import but is a little weird if you have multiple zip files that are being uploaded.  Especially because if you set it to the video default rate the fps value will be -1.  This may be okay and intended though.

Reply:
Done.